docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -519,3 +519,7 @@ Standard ONNX\* operators:
 | Upsample | No |
 | Where | No |
 | Xor | No |
+
+## Paddle\* Supported Layers
```

Comment:
This has included by LiuBo's PR: https://github.com/openvinotoolkit/openvino/pull/6687. So maybe we can leave it here?